maptivist/src/main/webapp/script.js

```diff
@@ -51,3 +51,70 @@ window.onclick = function(event) {
     form.style.display = "none";
   }
 }
+
+//GOOGLE OAUTH
+
+var GoogleAuth;
+  var SCOPE = 'https://www.googleapis.com/auth/drive.metadata.readonly';
```

Comment:
I think we want to use a scope that gets us the user's email address, so we can associate the user with our database.

This scope is for reading files in someone's Drive.

I think this one will work for us:
https://www.googleapis.com/auth/userinfo.email

I found it here:
https://developers.google.com/identity/protocols/oauth2/scopes#people

Reply:
Thank you, I just used that scope because it could read their data, I'll change Scopes immediately

Reply:
Fixed Scopes